src/test/java/hudson/plugins/clover/RatioTest.java

```diff
@@ -76,4 +76,48 @@ public void testGetPercentage() {
         assertEquals("1000/1000 => 100", 100, Ratio.create(1000,1000).getPercentage());
     }
 
+    /**
+     * Tests that {@link Ratio#getPcCovered()} correctly handles values near
+     * whole integers.  This is to make sure that it will match that of
+     * getPercentage()
+     */
+    public void testGetPcCovered() {
+        // Make sure zero is zero
+        assertEquals("0/10000     =>    0",    "0%", Ratio.create(    0, 10000).getPcCovered());
+        assertEquals("9/10000     =>    0",    "0%", Ratio.create(    9, 10000).getPcCovered());
+
+        assertEquals("99/10000    =>  0.9",  "0.9%", Ratio.create(   99, 10000).getPcCovered());
+
+        assertEquals("9009/10000  =>   90",   "90%", Ratio.create( 9009, 10000).getPcCovered());
+        assertEquals("9099/10000  => 90.9", "90.9%", Ratio.create( 9099, 10000).getPcCovered());
+
+        assertEquals("9909/10000  =>   99",   "99%", Ratio.create( 9909, 10000).getPcCovered());
+        assertEquals("9999/10000  => 99.9", "99.9%", Ratio.create( 9999, 10000).getPcCovered());
+
+        // still show 100
+        assertEquals("10000/10000 =>  100",  "100%", Ratio.create(10000, 10000).getPcCovered());
+    }
+
+    /**
+     * Tests that {@link Ratio#getPcUncovered()} correctly handles values near
+     * whole integers.  
+     * This needs to correctly contrast with getPcCovered()
+     */
+    public void testGetPcUncovered() {
+        // Nothing should be 100%
+        assertEquals("0/10000     =>  100",  "100%", Ratio.create(    0, 10000).getPcUncovered());
```

Comment:
I think we could give  a more meaningful messages in this method, as it's unclear how, e.g. "99/10000 translates to 99.1%" I was thinking about sth like:

"99/10000 = 0.99% covered = 99.01% uncovered = 99.1% rounded up"